keyboards/crbn/crbn.c

```diff
@@ -0,0 +1,9 @@
+#include "crbn.h"
+/* Encoder setting. only one encoder despite 4 possible spots */
+void encoder_update_kb(uint8_t index, bool clockwise) {
```

Comment:
This should/needs to call the user function, or should be the user function but set to be weak.

Reply:
I don't really understand what that means, I've tried looking up qmk user stuff but i don't see how it applies here

Reply:
To clarify this, for reference, is that the kb level functions need to call the user level functions.  

Additionally, the "weak" attribute allows the function to be replaced, without the compiler complaining.    This means that you can set a default, and allow the user replace the function easily.